charts/lakefs/templates/_arktika.tpl

```diff
@@ -0,0 +1,54 @@
+{{/*
+arktika resource full name
+*/}}
+{{- define "arktika.fullname" -}}
+{{- $name := include "lakefs.fullname" . }}
+{{- printf "%s-arktika" $name  | trunc 63 }}
+{{- end }}
+
+{{/*
+Common labels
+*/}}
+{{- define "arktika.labels" -}}
+helm.sh/chart: {{ include "lakefs.chart" . }}
+{{ include "arktika.selectorLabels" . }}
+{{- if .Chart.AppVersion }}
+app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
```

Comment:
I wonder - perhaps the version should be the mds image tag instead?
**(Feel free to not do it and resolve if you think otherwise)**